src/js/plugin/external/PluginHelper.js

```diff
@@ -1,27 +0,0 @@
-import PluginDispatcher from "./PluginDispatcher";
```

Comment:
So, you end up exactly in the same "jQuery"-global-scope. :) With the disadvantage of 	divergence.

Instead of a string for the constant we can use a unique Symbol, as you already suggested in the other PR.

We still just share the PluginDispatcher, these constants and helper on top are just convenient functions. The design is still the same and follows the proposal.